recipes/bcftools-snvphyl-plugin/meta.yaml

```diff
@@ -29,13 +30,8 @@ requirements:
 source:
   fn: bcftools-{{ version }}.tar.bz2
   url: https://github.com/samtools/bcftools/releases/download/{{ version }}/bcftools-{{ version }}.tar.bz2
-  md5: e43efe3df5f9c55acb4b48cee214d281
-  patches:
-    - b37b03e815cd4f25af70a57c08449f8c719dcc6a.patch # from https://github.com/samtools/bcftools/pull/645
-
+  sha256: 293010736b076cf684d2873928924fcc3d2c231a091084c2ac23a8045c7df982
 
 test:
   commands:
-    - bcftools -h
```

Comment:
@nsoranzo is there any reason you removed the tests?

Reply:
Yes, these are already tested in the main bcftools package, this now provides only the plugin.